api/resources_portal/urls.py

```diff
@@ -19,6 +22,11 @@
 router.register(r"materials", MaterialViewSet)
 router.register(r"organizations", OrganizationViewSet)
 
+search_router = DefaultRouter(trailing_slash=False)
+search_router.register(r"materials", MaterialDocumentView, basename="")
```

Comment:
Why are you setting `basename=""`?

Reply:
It can't infer the basename from the index as the document class in elasticsearch doesn't have a queryset with a name it could draw that from. It also apparently doesn't default to no basename, so I have to specify that.

Reply:
After discussion, modified basenames to be unique for each endpoint.